web/src/app/escalation-policies/PolicyStepForm.js

```diff
@@ -119,7 +119,7 @@ export default class PolicyStepForm extends React.Component {
 
     return (
       <FormContainer {...this.props}>
-        <Grid container spacing={3}>
+        <Grid container spacing={2}>
```

Comment:
![image](https://user-images.githubusercontent.com/29063970/61466002-c59d5600-a93e-11e9-8acc-9be101ec9bb4.png)
I think this change clicks the data badge at the top of the EP step form.

Reply:
good catch

Reply:
Ah, it looks like badges are being cut off at the top regardless of spacing. It must be clipping into the Dialog Title. I'll take note of that for a future issue to resolve